styles.css

```diff
@@ -1,54 +1,52 @@
 #title {
-    text-align: center;
+  text-align: center;
 }
 
 #survey-form {
-    text-align: center;
-    width: 40%;
-    background-color: grey;
```

Comment:
kindly would you please change the background-color to green as this way the text would be more readable

Reply:
great work but i suggest you to please change width to 50% so that content inside survey-form container would be more visible and eye catching